fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/MalformedRdfExceptionMapper.java

```diff
@@ -39,6 +39,10 @@
     public Response toResponse(final MalformedRdfException e) {
         final Link link = Link.fromUri(getConstraintUri(e)).rel(CONSTRAINED_BY.getURI()).build();
         final String msg = e.getMessage();
+
+        if (msg.contains("server-managed predicate")) {
```

Comment:
Not necessarily in this PR, but sometime, we should factor this string out into a constant in the appropriate class.